### DISCUSSION TITLE
陸版 7.0 9/27 上線了

### DISCUSSION THREAD

**Discussion by yoyoys** [GpointChen/FFXIVChnTextPatch-GP#474] [General]
這邊若要更新的話需要什麼資料？能幫什麼忙？

我是剛好喵到 https://github.com/thewakingsands/ffxiv-datamining-cn 才發現更新的

**Comment by HRFleur** [comment]
ACT好像已经更新了触发语言，7.0的副本和S这两天忽然不叫唤了xdd
